script.js

```diff
@@ -56,20 +61,22 @@ checkWordForLetter = (letter) =>{
 
 checkEndFailure = () => {
     if(gameInstance.attemptsRemaining === 0){
-        printToDom('You are terrible at this.  I guess try again...', '.failure', 'string')
+        printToDom('You are terrible at this. I guess try again...', '.failure', 'string')
         reloadGame()
     }
 }
 
 checkEndWin = () => {
     if(gameInstance.inProcessWordArray.indexOf('_') < 0){
         gameInstance.gamesWon += 1
+        document.querySelector('.word').classList.add('winning-word')
         let countdown = 4;
         let interval = setInterval(function(){
-            printToDom("Oh boy now you're going to get an ego " + --countdown, '.failure', 'string')
+            printToDom("Oh boy now you're going to get an ego. Next game in: " + --countdown, '.failure', 'string')
 
             if(countdown <= 0){
                 clearInterval(interval)
+                document.querySelector('.word').classList.remove('winning-word')
```

Comment:
see style.css. basically when you guess the word, the word gets bigger, bolder, and stronger. like you do after jazzercise.